tool/fetch-dart-sdk-sums.sh

```diff
@@ -43,13 +43,17 @@ for CHANNEL in $CHANNELS; do
       _arch='x64'
     fi
     _filename="dartsdk-linux-${_arch}-release.zip"
-    _url="$BASEURL/$CHANNEL/release/$VERSION/sdk/$_filename"
+    _urlNoFile="$BASEURL/$CHANNEL/release/$VERSION"
+    # Get the exact version from the json-formatted VERSION file:
+    _version=$(curl -fsSL "$_urlNoFile/VERSION" | awk -F: '/version/{print $2}' | sed 's/[", ]//g')
+    _url="$_urlNoFile/sdk/$_filename"
     curl -fsSLO $_url
     _checksum=$(shasum -a 256 $_filename)
     read -a _fname_arr <<< "${_checksum}" # Read in string output as array
     _checkonly="${_fname_arr%:*}" # Remove filename portion of checksum output
     printf "$(yellow "  DART_SHA256=\"$_fname_arr\";") $ENDING"
-    printf "$(yellow "  SDK_ARCH=\"$_arch\";;") $ENDING"
+    printf "$(yellow "  SDK_ARCH=\"$_arch\";") $ENDING"
+    printf "$(yellow "  DART_VERSION=\"$_version\";;") $ENDING"
```

Comment:
**issue:** This dumps numeric formatting characters instead of color coding, which has made updating even more difficult. Can this be changed?

That way, we could run the script to output to a file that we could then just copy/paste into this file. As is, I need to highlight and copy.